machine.go

```diff
@@ -576,26 +576,35 @@ func (m *Machine) setupLogging(ctx context.Context) error {
 	return nil
 }
 
-func captureFifoToFile(logger *log.Entry, fifoPath string, fifo io.Writer) error {
+func captureFifoToFile(logger *log.Entry, captureFifoPath string, fifo io.Writer) error {
```

Comment:
I think you actually do want `fifoPath`; it's the path of the fifo that we're reading.  You could change the `fifo io.Writer` to something like `w io.Writer` or `destination io.Writer`, which I think would be a lot more clear; the writer isn't a fifo and shouldn't be called one (we're reading a fifo, not writing one).

Reply:
Fixed!